assets/templates/passphrase_renew.html

```diff
@@ -39,10 +39,11 @@
     </svg>
     <div role="application">
       <main class="wizard">
-        <form id="renew-passphrase-form" method="POST" action="/auth/passphrase_renew" class="wizard-wrapper">
+        <form id="renew-passphrase-form" method="POST" action="/auth/passphrase_renew" class="wizard-wrapper" data-salt="{{.Salt}}">
           <h1 class="wizard-title u-mb-half u-mb-0-s u-mt-1-s">{{t "Passphrase renew Title"}}</h1>
           <div role="region" class="wizard-main">
             <input type="hidden" name="csrf_token" value="{{.CSRF}}" />
+            <input type="hidden" id="renew-password-iterations" name="iterations" value={{.Iterations}}>
```

Comment:
Le formulaire de création passe le nombre d'itérations via le `dataset` du formulaire. On ne peut pas faire la même chose ici ?

Reply:
Non, parce que l'on veut envoyer la valeur quand le formulaire est soumis.